ide/project.ant/src/org/netbeans/modules/project/ant/AntBasedProjectFactorySingleton.java

```diff
@@ -374,7 +374,7 @@ private Document loadProjectXml(File projectDiskFile) throws IOException {
             String msg = e.getMessage().
                     // org/apache/xerces/impl/msg/XMLSchemaMessages.properties validation (3.X.4)
                     replaceFirst("^cvc-[^:]+: ", ""). // NOI18N
-                    replaceAll("http://www.netbeans.org/ns/", ".../"); // NOI18N
+                    replace("http://www.netbeans.org/ns/", ".../"); // NOI18N
```

Comment:
FWIW, this is not the same before and after, but presumably bogus before.

Reply:
Correct, now the literal string **http://www.netbeans.org/ns/** is matched instead of http://www<any_character_here>netbeans.org/ns/